src/components/avatar/index.tsx

```diff
@@ -181,8 +195,8 @@ export class Avatar extends PureBaseComponent {
     };
   }
 
-  getRibbonStyle() {
-    const {size} = this.props;
+  getRibbonStyle(): StyleProp<ViewStyle> {
+    const {size = Avatar.defaultProps.size} = this.props;
```

Comment:
It feels kinda weird that we need to set the default here also. 
I assume TS throws an error here. 
I would check in the guild channel if someone know how to solve that. 
A lot of people know TS better than us. maybe there's a better solution

Reply:
This is a major issue with TS and React. A known issue with lots of weird and weirder solutions all very extravagant for this small issue we have here.
I asked around, it was suggested either `@ts-ignore` this, use constructor to initialize size and not use `defaultProps` at all...
The internet suggests even weirder gymnastics to solve this.
I don't feel. either suits this small `size` issue. IMO, we shouldn't impose unlogical changes in code to cater to TS. I can suggest leaving it like or ignoring it, or if you have a better solution...wdyt?

Reply:
Yes I agree. 
Anyway, I discussed with someone from the guild, he recommended changing the `size` prop to be not optional
This way we're telling it that we got the value covered with default props. 
Try it, it should work..

Reply:
Cool, seems to work 👍